app/views/play/cinematic/CinematicLankBoss.js

```diff
@@ -132,6 +136,26 @@ export default class CinematicLankBoss {
         })
       ]))
     }
+
+    const text = getText(dialogNode)
+    const animation = getSpeakingAnimation(dialogNode)
+    if (text && animation) {
+      const textLength = getTextAnimationLength(dialogNode)
+      const speaker = getSpeaker(dialogNode)
+      commands.push(new SequentialCommands([
+        new Sleep(Math.min(100, textLength)),
```

Comment:
Why sleep a min of 100 here?  Maybe a comment to explain?

Reply:
Great point!

Our Lank system has many concurrent systems running. And queuing an action immediately appears to just make the animation fail to play. This delay seems to allow some sort of setup to occur.